src/config.js

```diff
@@ -93,6 +120,28 @@ export function newConfig() {
 
   };
 
+  function hasGranularity(val) {
+    return Object.keys(GRANULARITY_OPTIONS).filter(option => val === GRANULARITY_OPTIONS[option]).length;
```

Comment:
`Object.keys(...).find()`?

Should be clearer & more efficient (since it short-circuits upon finding the first matching one)